src/test/java/com/jcabi/github/mock/MkCommentTest.java

```diff
@@ -43,6 +45,10 @@
  * @version $Id$
  */
 public final class MkCommentTest {
+    /**
+     * Example issue comment text.
+     */
+    private static final String COMMENT_TEXT = "what's up?";
```

Comment:
@cvrebert no need in this field. Just use different comments in places where we need them